src/main/io/displayport_max7456.h

```diff
@@ -20,7 +20,15 @@
 #include "config/parameter_group.h"
 #include "drivers/display.h"
 
-PG_DECLARE(displayPortProfile_t, displayPortProfileMax7456);
+typedef struct displayPortProfileMax7456_s {
```

Comment:
Why have you renamed this to `displayPortProfileMax7456_t`? The adjust, invert and brightness parameters are fairly generic and could apply to any OSD driver chip, so the name `displayPortProfile_t` is more appropriate. If/when we implement support for another type of OSD drive chip we can revisit the naming if there proves to be a conflict.